hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/snapshot/TestSnapshotChain.java

```diff
@@ -305,6 +307,34 @@ public void testChainFromLoadFromTable(boolean increasingTIme)
         () -> chainManager.nextGlobalSnapshot(snapshotID1));
   }
 
+  @ParameterizedTest
+  @ValueSource(ints = {0, 1, 2, 5, 10})
+  public void testSnapshotChainIterator(int numberOfSnapshots) throws IOException {
```

Comment:
Can we also add tests for `getOldestGlobalSnapshotId`. Just a line should be enough similar to `testChainFromLoadFromTable` and `testAddSnapshot`

Reply:
done